src/app/common/persistence.ts

```diff
@@ -11,7 +11,6 @@ export const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       cacheTime: PERSISTENCE_CACHE_TIME,
-      notifyOnChangeProps: ['data', 'error'],
```

Comment:
It appears from this the better solution might be to change this to `tracked`?
https://tkdodo.eu/blog/react-query-render-optimizations

Reply:
Not sure how recently this update on the blog is, but at the end of the Tracked Queries section it says,

> Update: Starting with v4, tracked queries are turned on per default in React Query, and you can opt out of the feature with notifyOnChangeProps: 'all'.

We're using v4 right? So should be the same as removing this line.

Reply:
Nice, I didn't see that update. 👍